### ISSUE TITLE
Organise files repo

### ISSUE THREAD

**Issue by TijmenW** [Kovadon/PokemonMOBA#9]
Reminder for self.

**Comment by TijmenW** [comment]
Are we going to merge the @runningfromlions and the resource folder?

**Comment by Kovadon** [comment]
I'd ask him, but I'd imagine so. Also sorry about that file that keeps getting committed, must be from me messing with it here and there.

**Comment by Kovadon** [comment]
He said go ahead and merge the folders

**Comment by TijmenW** [comment]
Now I think about it, is the assert folder not already a resource folder. 
Isn't it better if I/we put everything into the assert folder?

**Comment by Kovadon** [comment]
yeah, I was just doing that temporarily. Organize it how ever you feel is best to do

**Comment by TijmenW** [comment]
#16 organized a bit.

**Comment by TijmenW** [comment]
How can I, if needed, further organise it?

**Comment by Kovadon** [comment]
Currently it's looking good, was thinking maybe types of scripts should be organized (Ex: AI, GUI, other, etc.) and if you need to rename anything so that everything is relative, that'd work too.

**Comment by TijmenW** [comment]
The scripts folder is organized by @runningfromlions
